lib/services/book.ts

```diff
@@ -1,19 +1,66 @@
-import {prisma} from "@lib/services/db";
+import { prisma } from "@lib/services/db";
+import { getGoogleBooksData, GoogleBook } from "./googleBooksSearch";
+import { Book, Post, Course, Dept } from "@prisma/client";
+
+export type CourseWithDept = Course & {
+  dept: Dept;
+};
```

Comment:
I feel like this would make most sense if it was exported from the Course service

Reply:
Just saw I did not respond to this. Sure. Let's merge this first, and we can move it afterwards.